### ISSUE TITLE
Need some help with this issue 

### ISSUE THREAD

**Issue by DiptoChakrabarty** [CodeChefVIT/deployments#6]
![Screenshot (133)](https://user-images.githubusercontent.com/45638240/62153904-3a9b6300-b323-11e9-8a50-a3180aa53c67.png)

I have written the codes but dont know reason of error

**Comment by L04DB4L4NC3R** [comment]
Does this arise after the `npm install` command?

**Comment by DiptoChakrabarty** [comment]
Yes after adding the npm instsll command also this is occuring

**Comment by L04DB4L4NC3R** [comment]
This is because there in no `package.json` in the path that you have defined. Its better if you make a `Dockerfile` in the CET project, build and push the image. And then use that image in this repo

**Comment by DiptoChakrabarty** [comment]
I have made a custom docker image and the project works when made own docker container but when the container is made into an image and container is  deployed by port forwarding then website doesnt deploy on browser (localhost:port or ip of host:port ) .
Firewalld and Selinux has been stopped

**Comment by RishabhKodes** [comment]
> ![Screenshot (133)](https://user-images.githubusercontent.com/45638240/62153904-3a9b6300-b323-11e9-8a50-a3180aa53c67.png)
> 
> I have written the codes but dont know reason of error

Try using this code in your Dockerfile
WORKDIR /app
COPY package.json /app
RUN npm install
COPY . /app

**Comment by DiptoChakrabarty** [comment]
I have already done this it shows node_modules not installed or package.json not found
Its written in the dockerfile I uploaded in the repo

**Comment by RishabhKodes** [comment]
> I have already done this it shows node_modules not installed or package.json not found
> Its written in the dockerfile I uploaded in the repo

Don't use /root for routing. Just use /app

**Comment by L04DB4L4NC3R** [comment]
@DiptoChakrabarty Was this issue resolved?

**Comment by L04DB4L4NC3R** [comment]
@RishabhKodes I have left some review comments on #7. Resolve those and then you'll be good to go. Then this issue can be closed.
@DiptoChakrabarty you can try those out once the review is approved

**Comment by DiptoChakrabarty** [comment]
I have changed directory to app but there is no difference , in docker-compose  I have noticed 2 errors 
1) If i don't include  volume part (in my docker-compose file , first 2 lines in volume) , the error I get is express module missing  , node_modules not found 
2)If I include them then it shows package.json not found 
I included all the changes suggested 

Right now I am trying to make my own custom docker image and will deploy through that   will then work on the compose part

**Comment by L04DB4L4NC3R** [comment]
@DiptoChakrabarty Why are the following in this repo?

* Dockerfile
* package.json
* app.js

Meet me once, we will sort this out

**Comment by DiptoChakrabarty** [comment]
> @DiptoChakrabarty Why are the following in this repo?
> 
> * Dockerfile
> * package.json
> * app.js
> 
> Meet me once, we will sort this out

I was trying to do through docker compose at first so I  had them  in the repository , when do I come

**Comment by L04DB4L4NC3R** [comment]
I'm free now

**Comment by RishabhKodes** [comment]
Ports are different in all the files

**Comment by DiptoChakrabarty** [comment]
> Ports are different in all the files

Yes I had they got mixed up a bit as my port 3002 was occupied

**Comment by L04DB4L4NC3R** [comment]
Is it working now?

**Comment by DiptoChakrabarty** [comment]
> Is it working now?

Ya its working after a bit of modification , i ll push the changes
